packages/core/protocols/src/proto/dxos/config.proto

```diff
@@ -123,7 +130,8 @@ message Runtime {
     optional string public_url = 4 [(env) = "PUBLIC_URL"];
 
     optional BuildInfo build = 100;
-    optional google.protobuf.Struct env = 101;
+    // TODO: fix - is clashing with field option "env"
+    // optional google.protobuf.Struct env = 101;
```

Comment:
@dmaretskyi @wittjosiah I would need you help here - it seems like option `env` clashes with field `env`, e.g. go proto compiler is not happy with
```
Option field "(env)" is not a field or extension of message "FieldOptions".
```
Can we figure out naming there?

Reply:
for now i think you can rename the field extension to `env_var` and update its usages (i think only a few) since that would be non-destructive.

Reply:
if rename fixes the error - lets go for it. the error message seems a bit wierd, it misparses the env field for an option?

Reply:
It seems so - renaming does fix it;
Updated option name;
@dmaretskyi do we actually use it somehow, or it's mainly a declarative thing? seems like throughout the code we are reading those values directly/explicitly from `process.env`